app/selenium_ui/jira/modules.py

```diff
@@ -127,10 +127,11 @@ def __filer_epic(element):
 
             webdriver.find_element_by_id(ISSUE_TYPE_DROPDOWN).click()
             issue_elements = webdriver.find_elements_by_class_name("aui-list-item")
-            filtered_issue_elements = list(filter(__filer_epic, issue_elements))
-            rnd_issue_type_el = random.choice(filtered_issue_elements)
-            action = ActionChains(webdriver)
-            action.move_to_element(rnd_issue_type_el).click(rnd_issue_type_el).perform()
+            if issue_elements:
```

Comment:
When issue_elements has only one element, what would be a logic?

Reply:
We just move ahead. We don't need to choose an issue element because it already chose as default.

Reply:
if issue_elements - will return True, so logic will not move ahead.

Reply:
if issue_elements won't be empty it means that we have more than one issue element. Issue elements don't include element that we see without clicking on drop-down

Reply:
If I understand correctly, nothing will be returned (no aui-list-item), if we only have one issue type, right? So we just proceed without clicking on the dropdowns.

I think it would be better if we rename issue_elements to issue_elements_in_drodown to avoid confusion.

Reply:
I like Jun's suggestion. With current naming, it is not clear that `issue_elements` returns None in case of only one element.

Reply:
That's so true! I will rename it as Jun suggested